.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,116 @@
+---
+name: 🐛 Bug report
+description: Create a report to help us improve.
+title: "[BUG]: "
+labels: ["type/bug", "needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to create this bug report. To better help us investigate this bug, provide the following information.
+        Verify first that your issue is not already reported on GitHub
+        Also test if the latest release and devel branch are affected too
+        Complete *all* sections as described, this form is processed automatically
+  - type: textarea
+    id: Description
+    attributes:
+      label: Bug Description
+      description: Provide a clear and concise description about the bug.
+    validations:
+      required: true
+  - type: input
+    id: Product 
+    attributes:
+      label: Product Name
+      description: Product Name
+      placeholder: |
+        enterprise sonic,SFSS, SFS, etc.
+    validations:
+      required: true
+  - type: input
+    id: component
+    attributes:
+      label: Component or Module Name
+      description: Write the short name of the module, plugin, task or feature below, use your best guess if unsure
+      placeholder: sonic_bgp, sonic_ntp, sonic_interfaces, etc.
+    validations:
+      required: true
+  - type: input
+    id: ansible
+    attributes:
+      label: Ansible Collection Version
+      description: Enter the Ansible Collection version.
+      placeholder: |
+        Ansible 2.12
+    validations:
+      required: true
+  - type: input
+    id: Productv 
+    attributes:
+      label: Product Version
+      description: Enter the Product version. 
```

Comment:
It would be useful to specify the command that provides the version information in the description.

Reply:
Addressed as suggested.

Reply:
Resolved.